src/context/workout/index.tsx

```
@@ -0,0 +1,85 @@
+        workouts
+    });
+
+    const refreshPlans = async () => {
+        try {
+            const response = await fetch("/api/workout/plan")
+            const plans = await response.json() as WorkoutPlan[]
+            setModalStates((prevState) => ({
+                ...prevState,
+                plans
+            }))
+        } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)